$NetBSD: patch-tmake_lib_macosx-c++_tmake.conf,v 1.3 2015/01/26 13:35:37 pho Exp $

Make build honor PKGSRC_CFLAGS, otherwise linking against pkgsrc
libiconv will fail due to header/library mismatch.

--- tmake/lib/macosx-c++/tmake.conf.orig	2014-10-24 18:51:54.000000000 +0900
+++ tmake/lib/macosx-c++/tmake.conf	2015-01-27 13:48:44.000000000 +0900
@@ -11,7 +11,7 @@ TMAKE_CC		= cc
 TMAKE_CFLAGS		= -pipe 
 TMAKE_CFLAGS_WARN_ON	= -Wall -W -Wno-deprecated-declarations 
 TMAKE_CFLAGS_WARN_OFF	=
-TMAKE_CFLAGS_RELEASE	= -O3
+TMAKE_CFLAGS_RELEASE	= ${PKGSRC_CFLAGS}
 TMAKE_CFLAGS_DEBUG	= -g -fstack-protector
 TMAKE_CFLAGS_SHLIB	= -fPIC
 TMAKE_CFLAGS_YACC	= -Wno-unused -Wno-parentheses
